lua/nvim-paredit/init.lua

```diff
@@ -8,31 +8,50 @@ local M = {
   api = require("nvim-paredit.api"),
 }
 
-function M.setup(opts)
-  config.update_config(common.merge(defaults.defaults, opts))
+local function setup_keybingings(filetype)
+  local filetypes = config.config.filetypes
+  local keys = config.config.keys
 
+  if common.included_in_table(filetypes, filetype) then
+    keybindings.setup_keybindings({
+      keys = keys,
+      buf = 0,
```

Comment:
Does `0` always equal current buf, even when in the context of the `FileType` event autocmd? In other words is this the same as `event.buf`?

Reply:
0 means current buffer yes

Reply:
Ok, happy with either just want to confirm behaviour 👍🏻